en/docs/cli/ls.md

```diff
@@ -4,18 +4,32 @@ guide: docs_cli
 layout: guide
 ---
 
-<p class="lead">List installed packages.</p>
+<p class="lead">List all dependencies installed for the current working directory</p>
```

Comment:
That does exist as [`yarn global ls`](https://yarnpkg.com/en/docs/cli/global), I don't see Yarn adding `yarn ls --global`